rust/signer/src/lib.rs

```diff
@@ -223,6 +223,17 @@ fn history_seed_name_was_shown(seed_name: &str, dbname: &str) -> anyhow::Result<
         .map_err(|e| format!("{}", e))
 }
 
+fn export_key_info(dbname: &str) -> anyhow::Result<MKeysInfoExport, String> {
```

Comment:
To export only selected keys, which unique parameters of those keys will be needed here?
Is "`seedName`: String" unique? Because we have only MSeeds to show those screens on UI right now.

Reply:
@Dmitry-Borodin api has been updated with optional string array to handle that